src/lib/core/datetime/native-date-adapter.ts

```diff
@@ -202,14 +211,24 @@ export class NativeDateAdapter extends DateAdapter<Date> {
         this.getYear(date), this.getMonth(date), this.getDate(date) + days);
   }
 
-  getISODateString(date: Date): string {
+  toISODateString(date: Date): string {
     return [
       date.getUTCFullYear(),
       this._2digit(date.getUTCMonth() + 1),
       this._2digit(date.getUTCDate())
     ].join('-');
   }
 
+  fromISODateString(iso8601String: string): Date | null {
```

Comment:
Why isn't this captured as part of the normal parse function?

Reply:
This is more of a deserialization than parsing user input